specification/monitor/resource-manager/microsoft.insights/2017-05-01-preview/examples/listDiagnosticSettings.json

```diff
@@ -0,0 +1,47 @@
+{
+  "parameters" : {
+    "resourceUri" : "/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6",
+    "api-version" : "2017-05-01-preview"
+  },
+  "responses" : {
+    "200" : {
+      "headers" : {},
+      "body" : {
+        "value": [ 
+          {
+            "id":"/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6/diagnosticSettings/mysetting",
+            "type":"microsoft.logic/workflows",
+            "name":"mysetting",
+            "location":"eastus",
+            "tags":null,
```

Comment:
`tag` and `location` aren't allowed in `#/definitions/DiagnosticSettingsCategoryResource`.